src/Components/Article/ArticleDescription.jsx

```diff
@@ -37,11 +40,17 @@ function ArticleDescription() {
                     </p>
                     <p>
                         <b>Votes : </b>
-                        {article.votes}
+                        {votes}
                     </p>
                     <p>
                         <b>Comments made :</b> {article.total_comments}
                     </p>
+                    <Vote
+                        article={article}
+                        votes={votes}
+                        setVotes={setVotes}
+                        setArticle={setArticle}
+                    />
```

Comment:
the only prop you're actually using here is `setVotes` - remove these props when you stop using them